spec/electra/attestation.go

```diff
@@ -28,14 +28,11 @@ import (
 )
 
 // Attestation is the Ethereum 2 attestation structure.
-//
-//nolint:tagalign
 type Attestation struct {
-	AggregationBits bitfield.Bitlist `ssz-max:"131072" dynssz-max:"MAX_VALIDATORS_PER_COMMITTEE*MAX_COMMITTEES_PER_SLOT"`
+	AggregationBits bitfield.Bitlist `dynssz-max:"MAX_VALIDATORS_PER_COMMITTEE*MAX_COMMITTEES_PER_SLOT" ssz-max:"131072"`
 	Data            *phase0.AttestationData
-	Signature       phase0.BLSSignature `ssz-size:"96"`
-	// bitfield.Bitvector64 is an 8 byte array so dynamic sizing doesn't make sense.
-	CommitteeBits bitfield.Bitvector64 `ssz-size:"8"`
+	Signature       phase0.BLSSignature  `ssz-size:"96"`
+	CommitteeBits   bitfield.Bitvector64 `dynssz-size:"MAX_COMMITTEES_PER_SLOT/8" ssz-size:"8"`
```

Comment:
As per the comment, this is not a dynamic array but a Bitvector64 so of fixed size.  What would the `dynssz-size` tag change here?

Reply:
Yea, I think the `bitfield.Bitvector64` type is not ideal here.
The underlying base type is still a dynamic byte slice (`[]byte`), so the generic marshaller will just fill it with the given amount of bytes. That works fine even for repeated marshalling / unmarshalling, as long as the methods of `Bitvector64` are not gonna used excessively.

I've checked the dora codebase and I actually applied a workaround by not using the `Bitvector64.BitAt` method which fails with non expected array length, but using the `Bitvector64.BitIndices` method which works fine :D

Still definitely not ideal. The tag needs to be there anyway as otherwise the dynamic marshalling using the minimal preset fails. There are simply not 4 bytes available in a minimal block ssz stream.

Reply:
I've double checked, the removal of that tag broke the "weak support" for minimal preset blocks some time ago:
`failed to decode electra signed block contents\nfailed decoding field Message: failed decoding field Body: failed decoding field Attestations: incorrect size`

Works fine again with this branch :)   (we actually haven't used a minimal preset devnet for some time, so didn't really notice it yet)

Reply:
Okay, I think we're stuck with keeping this as a `Bitvector64` for mainnet compatibility, but understand the requirement for minimal.